public/data/VaibhavUpreti.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Vaibhav Upreti",
+  "type": "personal",
+  "bio": "Programmer | Developer | Coder | Student | Learner",
+  "avatar": "https://github.com/VaibhavUpreti.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/VaibhavUpreti",
+      "icon": "https://github.com/VaibhavUpreti.png"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vaibhav__upreti",
+      "icon": "https://twitter.com/vaibhav__upreti/photo"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```

Not sure if we can follow on LinkedIn 🤔 

```suggestion
      "name": "Connect with me on LinkedIn",
```